js/core.js

```diff
@@ -292,6 +292,7 @@ class UpdateHandler {
 
         if (oldVersion.isSameOrBefore("1.4.1")) {
             SyncedStorage.set("profile_steamid", SyncedStorage.get("profile_permalink"));
+            SyncedStorage.set("group_steamid", SyncedStorage.get("profile_permalink"));
```

Comment:
That setting is not related to the old one

Reply:
if u say so